.gitignore

```
@@ -1,2 +1,3 @@
-build
+build*
+output
```

```suggestion
build*/
output/
```

nit: ignore only directories?